### PR TITLE
Added Ammonium Nitrate (ANFO)

### PR DESCRIPTION
**Pull request by nicholast013**
## About The Pull Request
Just added Ammonium Nitrate an high explosive reagent that can be make using 1u EZ Nutrient, 1u Robust Harvest, 1u Left 4 Zed, 1u Welding Fuel the result is 4u of ammonium nitrate

## Why It's Good For The Game
More explosive chemical more fun!!

## Changelog
:cl:
add: Added Ammonium Nitrate as high explosive chemical and a fertilizer
/:cl:

### CODE REVIEW

**Comment by TheChosenEvilOne** [comment]
pipe bomb time

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm [hunk 1]
```diff
@@ -119,6 +119,15 @@
 	e.start()
 	holder.clear_reagents()
 
+/datum/reagent/anfo
+	name = "Ammonium Nitrate"
+	description = "High explosive fertilizer"
```
Comment (TheChosenEvilOne) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm:124]: you should actually make it a fertilizer, see saltpetre and the nutriments in `modules/reagents/chemistry/reagents/other_reagents.dm`

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm [hunk 2]
```diff
@@ -152,6 +152,16 @@
...
 
+/datum/chemical_reaction/anfo
+	results = list(/datum/reagent/anfo = 3)
+	required_reagents = list(/datum/reagent/plantnutriment/eznutriment = 1, /datum/reagent/plantnutriment/left4zednutriment = 1, /datum/reagent/plantnutriment/robustharvestnutriment = 1, /datum/reagent/fuel = 1)
+
+/datum/chemical_reaction/reagent_explosion/anfo_explosion
+	required_reagents = list(/datum/reagent/anfo = 1)
+	required_temp = 474
+	strengthdiv = 5
+	modifier = 10
```
Comment (TheChosenEvilOne) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm:163]: these should be moved below the gunpowder_explosion/on_reaction proc.

**Comment by stanalbatross** [comment]
more descriptive changelog needed

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm (hunk 1 above)
Comment (nicholast013) [reply code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm:124]: ah
forgot about that

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm (hunk 2 above)
Comment (nicholast013) [reply code/modules/reagents/chemistry/recipes/pyrotechnics.dm:163]: thank you i didnt see the proc

**Comment by Pugie** [comment]
o shit we ghetto now

**Comment by Pugie** [comment]
1u gibs

**Pull request merged**